hi-michelleprimiani/levelup-server
# Models and database created

## Models 

- Four model modules were created. 

1. event.py 
 ```
Table Event {
  id int pk
  organizer_id int
  game_id int
  name varchar
  location varchar
  time datetime
  date datetime
```

2. game.py 
```
Table Game {
  id int pk
  name varchar
  gametype_id int
  user_id int
  description varchar
  difficultly int
  duration float
  num_of_players int
}
```

3. gametype.py
```
Table GameType {
  id int pk
  description varchar
}
```

4. eventgamer.py
```
Table EventGamer {
  id int pk
  user_id int
  event_id int
}
```


## Migrations 

- Each module has been successfully migrated 

## How To Test 
- Pull ```mp-init``` branch down
- npm run dev
- go to db.sqlite3
- check that all fields of the table exist
